src/components/wallet/events/render.tsx

```diff
@@ -1,33 +1,103 @@
-import { h } from '@stencil/core'
-import { has as loHas } from 'lodash-es'
+import { h } from "@stencil/core";
+import { has as loHas } from "lodash-es";
 
 export default function render() {
+  const loggedInContent = () => [
```

Comment:
I pulled out the contents of the body into the loggedIn/OutContent functions to keep things clearer vs putting so much in a ternary operator in the body.